frontend/src/types/common.ts

```diff
@@ -951,3 +951,10 @@ const makeEmpty = (type: ResourceType) => {
   }
 };
 export const empty = makeEmpty as ResourceMaker;
+
+export const routeHelpNameMap = {
```

Comment:
let's move this to a separate file and comment on the wiring logic.

The goal is the DevMar team can add documentation independently.